<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTagWidget</class>
 <widget class="QWidget" name="AutoTagWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chbEnable">
     <property name="text">
      <string>&amp;Enable automatic #tagging</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lblAdd">
       <property name="text">
        <string>&amp;Add words:</string>
       </property>
       <property name="buddy">
        <cstring>lineWord</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineWord"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnAdd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset resource="../../qtwitter-app/res/resources.qrc">
         <normaloff>:/icons/add_48.png</normaloff>:/icons/add_48.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset resource="../../qtwitter-app/res/resources.qrc">
         <normaloff>:/icons/cancel_48.png</normaloff>:/icons/cancel_48.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="viewWords">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qtwitter-app/res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
